js/viz/docs/doccharts.js

```diff
@@ -1623,6 +1659,34 @@ var dxChart = {
             text: undefined
         },
         /**
+        * @name dxchartoptions_argumentaxis_workdaysonly
+        * @publicName workdaysOnly
+        * @type boolean
+        * @default false
+        */
+        workdaysOnly: false,
+        /**
+        * @name dxchartoptions_argumentaxis_workweek
+        * @publicName workweek
+        * @type Array<number>
+        * @default [1, 2, 3, 4, 5]
+        */
+        workweek: [1, 2, 3, 4, 5],
```

Comment:
I see that in [other parts of devexteme](https://js.devexpress.com/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/views/) we use camel case notation for this word.